satpy/tests/test_dataset.py

```diff
@@ -167,8 +167,25 @@ def test_combine_arrays(self):
               ]
         assert "quality" not in combine_metadata(*dts6)
 
+    def test_combine_lists(self):
+        """Test combine metadata with different types of lists."""
+        from satpy.dataset.metadata import combine_metadata
+        metadatas = [
+            {'prerequisites': [1, 2, 3, 4]},
+            {'prerequisites': [1, 2, 3, 4]},
+        ]
+        res = combine_metadata(*metadatas)
+        assert res['prerequisites'] == [1, 2, 3, 4]
+
+        metadatas = [
```

Comment:
Could you make this a new method, and rename both to be more specific? eg test_combine_identical_lists vs test_combine_different_lists.

Which trigger the question: what happens if the two lists have the same size but different values? (I know the answer, but it may be worth having a test for that case so that we are clear on that)

Reply:
Sounds good. Will do.

Reply:
Done.